### ISSUE TITLE
Apple M1 compatibility problem with .NET 6.0

### ISSUE THREAD

**Issue by bjsmith** [MikhailArkhipov/vscode-r#90]
I'm trying to figure out the right prerequisites for the following system:

- Apple M1 Pro
- macOS Monterey v12.4
- Visual Studio Code Version 1.72.0

VSCode instructions say it supports "Mac OS 11.2+ Intel or ARM with Rosetta 2 (Rosetta for .NET language server, R 4.1+ ARM64, .NET 5 x64)"

The prerequisites are:

 - [.NET 5.0 Runtime](https://www.microsoft.com/net/download) - x64 on Mac even with ARM
 - [R distribution (64-bit only)](https://cloud.r-project.org/)
 - With Apple M1 silicon and R 4.1+ please use ARM64 package.
 
This is quite confusing. The first point says use x64 on Mac even with ARM; the third bullet point says to "please use ARM64 package". On reflection I guess the instruction to use ARM64 applies to the R distribution? This could be clearer.

**Comment by bjsmith** [comment]
Update:

I deleted VS 6.0, and installed the dotnet sdk 5.0 osx-x64. 

On successful installation the pkg displays the message

> The following was installed at /usr/local/share/dotnet 
> 	•	.NET SDK 5.0.408
> 	•	.NET Runtime 5.0.17
> 	•	ASP.NET Core Runtime 5.0.17
> 

Then, when trying to run R Tools, I get:

```
ou must install or update .NET to run this application.

App: /Users/benjaminsmith/.vscode/extensions/mikhail-arkhipov.r-0.0.28/ls/Microsoft.R.LanguageServer.dll
Architecture: arm64
Framework: 'Microsoft.NETCore.App', version '5.0.0' (arm64)
.NET location: /usr/local/share/dotnet/

No frameworks were found.

Learn about framework resolution:
https://aka.ms/dotnet/app-launch-failed

To install missing framework, download:
https://aka.ms/dotnet-core-applaunch?framework=Microsoft.NETCore.App&framework_version=5.0.0&arch=arm64&rid=osx.12-arm64
[Info  - 5:07:32 PM] Connection to server got closed. Server will restart.
You must install or update .NET to run this application.


```

**Comment by MikhailArkhipov** [comment]
Should be resolved in 0.30. Built for .NET 6, Mac OS 13, R 4.2 (ARM only), dropping Intel on Mac.
